.travis.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       script:
         - npm run build
         - xvfb-run -a npm run test:prod:travis
+        - npm prune --production
```

Comment:
Does this do anything? Im pretty sure this is the end of the stage?

Reply:
> it's also important for when jenkins is down that we can trust the travis build

Reply:
>Data persistence between stages and jobs
>
>It is important to note that jobs do not share storage, as each job runs in a fresh VM or container. If your jobs need to share files (e.g., using build artifacts from the “Test” stage for deployment in the subsequent “Deploy” stage), you need to use an external storage mechanism such as S3 and a remote scp server.

https://docs.travis-ci.com/user/build-stages/#data-persistence-between-stages-and-jobs

Reply:
It doesn't do anything at all at the moment, I think it should be done between build and the tests to mimic the live deploy and prune what won't be there in production.

Reply:
@thekp I think your `npm prune --production` needs to be moved up a line in between build and `npm run test:prod:travis`

But I think benjamins doing this here https://github.com/bbc/simorgh/pull/2905/files